stack.yaml

```diff
@@ -6,3 +6,6 @@ packages:
 
 nix:
   shell-file: scripts/nix/stack-shell.nix
+
+extra-deps:
```

Comment:
You should move this dep into the snapshot if it's going to affect everything built with the snapshot!

Reply:
Done!